File: PHASE1/ANGULAR/my-first-angular-project/src/app/model/product.ts
export class Product {

    public id: number = 0;
    public name: string = '';
    public price: number = 100;

    constructor(id: number, name: string, price: number) {
        this.id = id;
        this.name = name;
        this.price = price;
    }

}

File: PHASE3/JSPDemo/src/main/java/com/sl/Product.java
package com.sl;

import java.io.Serializable;

// Following is a Java bean 
public class Product implements Serializable{
	
	private String name;
	private float price;
	
	public Product() {
		
	}
	
	public Product(String name, float price) {
		super();
		this.name = name;
		this.price = price;
	}


	public String getName() {
		return name;
	}


	public void setName(String name) {
		this.name = name;
	}


	public float getPrice() {
		return price;
	}


	public void setPrice(float price) {
		this.price = price;
	}	

}
